### PR TITLE
Do not clutter data/ directory with grants

### PR DESCRIPTION
**Pull request by mjgiarlo**
Put extracted and transformed grants in subdirectories like the publication ETL is doing.

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
 # Grant ETL
 every 2.months do
   exe 'exe/transform call StanfordPeopleList -i data/researchers.ndj > data/researchers.csv ' \
-    '&& exe/grants load -s 3 -i data/researchers.csv '
+    '&& exe/grants load -s 3 -i data/researchers.csv -d data/raw/grants -o data/grants'
```
Comment (jcoyne) [inline config/schedule.rb]: do you need to make a `data/raw/grants/.keep` file and add an entry into `.gitignore`?

Comment (mjgiarlo) [reply config/schedule.rb]: I don't know. I hand-added these to `./shared/` on the prod server, but letting the repo handle this seems more foolproof. Change incoming.

File: .gitignore [hunk 2]
```diff
@@ -14,5 +14,10 @@ config/honeybadger.yml
 config/settings.local.yml
 config/settings/*.local.yml
 warnings*.log
+/data/
```
Comment (mjgiarlo) [inline .gitignore]: This makes sure all the other stuff in `data/` do not clutter git tracking.

File: .gitignore [hunk 3]
```diff
@@ -14,5 +14,10 @@ config/honeybadger.yml
...
 !/data/.keep
+!/data/grants/.keep
```
Comment (mjgiarlo) [inline .gitignore]: These are the four directories used by `config/schedule.rb`

**Comment by mjgiarlo** [comment]
@jcoyne 	take another gander?

File: .gitignore [hunk 4]
```diff
@@ -14,5 +14,10 @@ config/honeybadger.yml
 config/settings.local.yml
 config/settings/*.local.yml
 warnings*.log
+/data/
 !/data/.keep
```
Comment (jcoyne) [inline .gitignore]: You could probably delete this file and this line now.

Comment (mjgiarlo) [reply .gitignore]: Done. Pushing now.

File: config/schedule.rb (hunk 1 above)
Comment (mjgiarlo) [reply config/schedule.rb]: Done.

**Pull request merged**